### PR TITLE
Allow user to set target version

### LINKED ISSUE #4: Autohupr should support auto-updating to a target release instead of latest
This block should have an environment variable that will contain the target OS version. When the target version is greater (or different) than the running version.

The user will be able to trigger an update by modifying the variable, either in a fleet or in a device.

### PR DESCRIPTION
**Pull request by jaomaloy**
This resolves #4

### CODE REVIEW

**Comment by klutchell** [comment]
Running `npm run lint-fix` should resolve the error `Error: File src/main.ts hasn't been formatted with prettier`

File: src/main.ts [hunk 1]
```diff
@@ -62,11 +65,15 @@ const getTargetVersion = async (
 ): Promise<string | null> => {
 	return await balena.models.os
 		.getSupportedOsUpdateVersions(deviceType, deviceVersion)
-		.then((versions) => {
-			if (versions.recommended) {
-				return versions.recommended;
+		.then((osUpdateVersions) => {
+			if (userTargetVersion === '') {
+					return null;
```
Comment (klutchell) [inline src/main.ts]: Should we print some kind of error to indicate `HUP_TARGET_VERSION` must be set?

Alternately, we could add ` ENV HUP_TARGET_VERSION recommended` in the Dockerfile and document what values are supported?

Comment (jaomaloy) [reply src/main.ts]: doesn't setting ` ENV HUP_TARGET_VERSION recommended` make it the default? We said [here](https://github.com/balena-labs-projects/autohupr/pull/5/files/bcd2e2b32ece5905e43af099b89420e83c4480b1#r985958258) that we make doing nothing the default case if no value is specified.

Comment (klutchell) [reply src/main.ts]: I forgot about that discussion! If that's the intended behaviour then it looks good to me!

File: src/main.ts [hunk 2]
```diff
@@ -62,11 +65,25 @@ const getTargetVersion = async (
...
-			} else {
+		.then((osUpdateVersions) => {
+			if (userTargetVersion === '') {
+				console.log(
+					'HUP_TARGET_VERSION must be set to perform automatic updates.',
+				);
 				return null;
+			} else {
+				if (
+					userTargetVersion === 'recommended' ||
```
Comment (klutchell) [inline src/main.ts]:
```suggestion
					['recommended','latest'].includes(userTargetVersion)
```
I sometimes find this syntax easier to read, but it's up to you.

**Comment by klutchell** [comment]
lgtm

**Pull request merged**